src/Microsoft.Tye.Core/Microsoft.Tye.Core.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Why downgrade these?

Reply:
The TFMs were upgraded to support "net5.0" projects. Now that's no longer needed so I've reverted them to netcoreapp3.1 which is LTS.

Reply:
But 5.0 is cool and new.